scripts/update-dependencies.ps1

```diff
@@ -32,9 +32,15 @@ $env:PATH = "$env:DOTNET_INSTALL_DIR;$env:PATH"
 
 $appPath = "$PSScriptRoot\update-dependencies"
 
-# Restore the build scripts
-Write-Host "Restoring Build Script projects..."
-pushd $PSScriptRoot
+# Restore the build_projects
+Write-Host "Restoring Microsoft.DotNet.Cli.Build.Framework..."
+pushd $PSScriptRoot\..\build_projects\Microsoft.DotNet.Cli.Build.Framework
```

Comment:
I am leaning towards moving update-dependencies into the `build_projects` folder to fix this kind of problem - it will live next to Microsoft.DotNet.Cli.Build.Framework again.

Any objections?  @brthor